src/CssReset/index.js

```diff
@@ -0,0 +1,166 @@
+import React from 'react'
+
+function CssReset() {
+    return (
+        <style jsx global>{`
+            /* http://meyerweb.com/eric/tools/css/reset/
```

Comment:
I think we can move that outside of the style tag, no need for styled-jsx do strip that?

Reply:
Agreed.